lib/Raven/Client.php

```diff
@@ -805,7 +805,9 @@ protected function get_user_data()
             $user = array(
                 'id' => session_id(),
             );
-            if (!empty($_SERVER['REMOTE_ADDR'])) {
+            if (!empty($_SERVER['HTTP_X_FORWARDED_FOR'])) {
```

Comment:
this is incorrect and unsafe as a default -- any user can set HTTP_X_FORWARDED_FOR, not just your proxy, and you need to explicit enable this kind of configuration in the SDK by saying "my webserver is configured to send this"

Reply:
Can this configuration be added to the SDK?

Isn't using REMOTE_ADDR also unsafe for the same reason?

At the moment this feature (recording of the client IP) is broken as soon as your app is behind a proxy for sentry.

Reply:
We generally assume that a rewrite happens in a middleware so `REMOTE_ADDR` points to the right thing. I think some sdks also pull from framework's remote addr value which might be rewritten by a middleware.

Reply:
How can this be resolved then please, i'm using symfony framework.